prov/efa/src/rxr/rxr_init.c

```diff
@@ -668,6 +680,14 @@ EFA_INI
 			"Set the number of EFA completion entries to read for one loop for one iteration of the progress engine. (Default: 50)");
 	fi_param_define(&rxr_prov, "shm_cq_read_size", FI_PARAM_SIZE_T,
 			"Set the number of SHM completion entries to read for one loop for one iteration of the progress engine. (Default: 50)");
+	fi_param_define(&rxr_prov, "inter_max_long_msg_size", FI_PARAM_INT,
```

Comment:
Can you do this in this PR? Or at least before 1.10 is cut?

Reply:
I think man page change need to be a separate PR, and we need to consult many other people on the exact wording because it is customer facing. It definitely need to be done before 1.10 is released, but it can be after rc1. 

I have added this to the issue I created.

Reply:
Okay.